File: Python/Human.py
class Human:

    # Kelas Human di sini berperan sebagai:
    # - Parent Class dari Kelas Sivitas Akademik 
    # - Grandparent Class dari Kelas Mahasiswa dan Dosen

    # ATRIBUT

    __NIK = ""
    __nama = ""
    __jenis_kelamin = ''

    # METHODS
    # Constructor
    def __init__(self, NIK="", nama="", jk=''):
        self.__NIK = NIK
        self.__nama = nama
        self.__jenis_kelamin = jk

    # Setter and Getter
    def setNIK(self, NIK):
        self.__NIK = NIK
    
    def setNama(self, nama):
        self.__nama = nama

    def setJenisKelamin(self, jk):
        self.__jenis_kelamin = jk

    def getNIK(self):
        return self.__NIK
    
    def getNama(self):
        return self.__nama
    
    def getJenisKelamin(self):
        return self.__jenis_kelamin
    
    

File: CPP/Program/human.cpp
#include "header.hh"

// == KELAS HUMAN ==

// Constructors
Human::Human()
{
    NIK = "";
    nama = "";
    jenis_kelamin = '-';
}

Human::Human(string nik, string nama, char jk)
{
    this->NIK = nik;
    this->nama = nama;
    this->jenis_kelamin = jk;
}

// Getter and Setter
string Human::getNIK()
{
    return this->NIK;
}
string Human::getNama()
{
    return this->nama;
}
char Human::getJenisKelamin()
{
    return this->jenis_kelamin;
}
void Human::setNIK(string nik)
{
    this->NIK = nik;
}
void Human::setNama(string nama)
{
    this->nama = nama;
}
void Human::setJenisKelamin(char jk)
{
    this->jenis_kelamin = jk;
}

// Destructor
Human::~Human() {}